bouquet/io.py

```
@@ -159,6 +170,84 @@ def add_entry(coords, atoms: Atoms, energy: float) -> None:
+            "e_eval": e_eval,
+            "e_best": e_best,
+            "mu_min": cert.get("mu_min"),
+            "alpha_max": cert.get("alpha_max"),
+            "n_calls": n_calls,
+            "wall_s": wall_s,
+            "t_select": cert.get("t_select"),
+            "t_eval": cert.get("t_eval"),
+            "t_gp_fit": cert.get("t_gp_fit"),
+            "t_acq": cert.get("t_acq"),
+        }
+        for col, val in zip(lb_cols, cert.get("lb", [])):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)